config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleMetadata.java

```diff
@@ -138,9 +139,10 @@ public Builder description(final String description) {
             return this;
         }
 
-        public Builder reusableAndRedeployable(boolean reusable, boolean redeployable) {
+        public Builder reusableRedeployableAndHasRouting(boolean reusable, boolean redeployable, boolean hasRouting) {
```

Comment:
accepting multiple booleans over a single method is not a good practice. 
Think of providing separate methods.

Reply:
@graju256 done!